data/mods/MMA/modinfo.json

```
@@ -0,0 +1,11 @@
+[
+  {
+    "type": "MOD_INFO",
+    "ident": "MMA",
+    "name": "Mythical Martial Arts",
+    "authors": [ "Photoloss", "C:DDA main repo" ],
```

I'm guessing `C:DDA main repo` is meant for anyone to add, edit or otherwise modify?

I'd suggest removing the line and adding a second line:
```suggestion
    "authors": [ "Photoloss" ],
    "maintainers": [ "Anyone" ],
```

Hopefully makes it understood that anyone can make changes to it. I'm not 100% sure on the guidelines for stuff like this, but typically any "official" mods are "anyone and everyone"'s responsibility, so it might be moot.

---

The main point from my side was that I am not actually the "original author" of the code for the martial arts. If I were to post chapters from *Alice in Wonderland* on a blog for example the credited "author" should be Lewis Carroll and not myself as the blog owner.

---

You can remove `authors` and `maintainers` nodes - they aren't necessary.